guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -4033,7 +4044,8 @@ WEBVTT
 [Stephen Frost]: in their importance more at the expense of God.
 
 02:23:09.040 --> 02:23:09.600
-[J Jay Couey]: Is that correct?
+[
+stephen Frost]: Is that correct?
```

Comment:
Probably an accidental enter here. This should be joined with the line above and start w/ a capital S.

Reply:
Agreed.